src/components/TableHeadCell.js

```diff
@@ -197,37 +203,38 @@ const TableHeadCell = ({
       onMouseDown={closeTooltip}
       {...otherProps}>
       {options.sort && sort ? (
-        <span
-          role="button"
-          onKeyUp={handleKeyboardSortInput}
-          onClick={handleSortClick}
-          className={classes.toolButton}
-          data-testid={`headcol-${index}`}
-          ref={isDraggingEnabled() ? dragRef : null}
-          tabIndex={0}>
+        <span className={classes.contentWrapper}>
           <Tooltip
             title={getTooltipTitle()}
-            placement={'bottom-start'}
+            placement="bottom-center"
```

Comment:
For what reason?

Reply:
This was to better line up the tooltip with the new Material-UI button added for the focus/hover effects. I can revert this if we want to leave default alignment to bottom-start.

Reply:
It looks good to me too